tools/src/main/java/com/nedap/archie/archetypevalidator/ArchetypeValidator.java

```diff
@@ -170,6 +172,14 @@ public ValidationResult validate(Archetype archetype, FullArchetypeRepository re
         if(result.passes() || settings.isAlwaysTryToFlatten()) {
             try {
                 Archetype flattened = new Flattener(repository, combinedModels).flatten(archetype);
+
+                try {
+                    OperationalTemplate operationalTemplate = (OperationalTemplate) new Flattener(repository, combinedModels).createOperationalTemplate(true).flatten(archetype);
+                    extraRepository.addExtraOperationalTemplate(operationalTemplate);
+                } catch (Exception e) {
+                    //this is probably an error in an included archetype, so ignore it here
+                    //the other archetype will not validate
```

Comment:
Can you catch a specific exception or at least write a log message?

Reply:
logging is not a good idea in a library generally. Perhaps we could add ValidationMessage with category WARNING/INFO. Otherwise, this failing is not directly a problem with this archetype, so it should not fail validation.